types/react-instantsearch/types/index.d.ts

```diff
@@ -0,0 +1,267 @@
+export interface AlgoliaError {
```

Comment:
Recommend that you move types to respective modules instead of holding them here. Or keep in index.d.ts to avoid users having to export it from submodule.